src/nativeapm/elements/inputs/select.ts

```diff
@@ -0,0 +1,158 @@
+/// <reference path="../../references.ts" />
+
+/**
+ * ProcessOut module/namespace
+ */
+module ProcessOut {
+  /**
+   * ProcessOut Native APM class for handling select input
+   */
+  export class NativeApmSelectInput implements INativeApmInput {
+    /**
+     * Native APM form element
+     * @type {HTMLInputElement}
+     */
+    inputElement: HTMLSelectElement;
+
+    /**
+     * Native APM input data
+     * @type {NativeApmInputData}
+     */
+    inputData: NativeApmInputData;
+
+    /**
+     * Native APM input label element
+     * @type {HTMLElement}
+     */
+    inputLabel: HTMLElement;
+
+    /**
+     * Native APM error message element
+     * @type {HTMLElement}
+     */
+    errorMessageElement: HTMLElement;
+
+    /**
+     * Theme of the Native APM widget
+     * @type {NativeApmThemeConfigType}
+     */
+    theme: NativeApmThemeConfigType;
+
+    /**
+     * Native APM Input constructor
+     */
+    constructor(
+      inputData: NativeApmInputData,
+      theme: NativeApmThemeConfigType
+    ) {
+      this.theme = theme;
+      this.inputData = inputData;
+      this.inputElement = this.createInputElement();
+      this.inputLabel = this.createInputLabel();
+      this.errorMessageElement = this.createErrorMessageElement();
+    }
+
+    /**
+     * This function returns the input element
+     */
+    public getInputElement() {
+      const inputWrapper = document.createElement('div');
+
+      StylesUtils.styleElement(inputWrapper, this.theme.form.inputs.wrapper);
+
+      inputWrapper.appendChild(this.inputLabel);
+      inputWrapper.appendChild(this.inputElement);
+      inputWrapper.appendChild(this.errorMessageElement);
+
+      return inputWrapper;
+    }
+
+    /**
+     * This function validates the input
+     */
+    public isValid() {
+      return true;
```

Comment:
It will always be true since we have predefined options from the API, but the interface which I provided requires that so I thought that it will be the simplest solution